src/components/inputs/money-input/money-input.js

```diff
@@ -311,16 +312,40 @@ const getCurrencyDropdownName = name =>
 
 const MoneyInput = props => {
   const intl = useIntl();
-  const [currencyHasFocus, setCurrencyHasFocus] = React.useState(false);
-  const [amountHasFocus, setAmountHasFocus] = React.useState(false);
+  const [
+    currencyHasFocus,
+    // eslint-disable-next-line no-unused-vars
+    toggleCurrencyHasFocus,
+    setCurrencyHasFocusOn,
+    setCurrencyHasFocusOff,
+  ] = useToggleState(false);
+  const [
+    amountHasFocus,
+    // eslint-disable-next-line no-unused-vars
```

Comment:
this is a sort of annoying part of using hooks 🤔

Reply:
I think you can do this:
```
const [amountHasFocus, ,setAmountHasFocusOn, setAmountHasFocusOff] = useToggleState(false);
```
to ignore the returned `toggleCurrencyHasFocus`, since you don't need it.
[source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Ignoring_some_returned_values)